tests/Jobs/SendEmptyWikibaseNotificationsJobTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Tests\Jobs;
+
+use App\Jobs\SendEmptyWikibaseNotificationsJob;
+use App\Notifications\EmptyWikibaseNotification;
+use App\User;
+use App\Wiki;
+use App\WikiManager;
+use Illuminate\Contracts\Queue\Job;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\Notification;
+use Tests\TestCase;
+
+class SendEmptyWikibaseNotificationsJobTest extends TestCase
```

Comment:
Some additional scenarios we might want to cover in the tests:
- A wiki that has seen activity, but then was abandoned does not receive a notfication
- An empty wiki receives the notification only once
- Active wikis do not receive notifications